bb_wrapper/wrapper/bb.py

```diff
@@ -12,10 +12,10 @@ class BaseBBWrapper(RequestsWrapper):
     """
 
     BASE_SCHEMA = "https://"
-    BASE_SUBDOMAIN = "api"
-    BASE_SANDBOX_ADDITION = ".sandbox"
     BASE_PROD_ADDITION = ""
     BASE_DOMAIN = ".bb.com.br"
```

Comment:
remover esses outros campos tbm!

```Suggestion

```

Reply:
Tem que deixar com if/else se for de sandbox ou não!

Mas pode deixar fixa sim.

- `https://oauth.sandbox.bb.com.br/oauth/token`
- `https://oauth.bb.com.br/oauth/token`